docs/useInput.md

```diff
@@ -99,14 +99,6 @@ const LatLngInput = props => {
     );
 };
 ```
-
-**Reminder:** [react-hook-form's `formState` is wrapped with a Proxy](https://react-hook-form.com/docs/useformstate/#rules) to improve render performance and skip extra computation if specific state is not subscribed. So, make sure you deconstruct or read the `formState` before render in order to enable the subscription.
-
-```js
-const { isDirty } = useFormState(); // ✅
-const formState = useFormState(); // ❌ should deconstruct the formState      
-```
-
```

Comment:
I believe we should keep this one. @fzaninotto?

Reply:
It's true that the reminder is mentioned twice in this chapter. I'd create a dedicated section explaining how to read the form state, and removing the reminder from the two sections avout TextField and Select.

Reply:
@djhi Do you want to do it or should I? Rock paper scissors?

Reply:
:rock: That's not really related to your PR so if you don't, we'll open a new PR for that.

Reply:
I'm on it then, after all, I broached it by removing the repeated section.